### PR TITLE
Convert lib/actions/help.coffee to javascript

### PR DESCRIPTION
**Pull request by Page-**
Change-type: patch

Resolves: #   
Change-type: major|minor|patch   
Depends-on: <url>   
See: <url>   

---
Please check the CONTRIBUTING.md file for relevant information and some
guidance. Keep in mind that the CLI is a cross-platform application that runs
on Windows, macOS and Linux. Tests will be automatically run by balena CI on
all three operating systems, but this will only help if you have added test
code that exercises the modified or added feature code.

Note that each commit message (currently only the first line) will be
automatically copied to the CHANGELOG.md file, so try writing it in a way
that describes the feature or fix for CLI users.

If there isn't a linked issue or if the linked issue doesn't quite match the
PR, please add a PR description to explain its purpose or the features that it
implements. Adding PR comments to blocks of code that aren't self explanatory
usually helps with the review process.

If the PR introduces security considerations or affects the development, build
or release process, please be sure to highlight this in the PR description.

### CODE REVIEW

File: lib/actions/help.js [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+const general = function(_params, options, done) {
+	console.log('Usage: balena [COMMAND] [OPTIONS]\n');
+	console.log(messages.reachingOut);
+	console.log('\nPrimary commands:\n');
+
+	// We do not want the wildcard command
+	// to be printed in the help screen.
+	const commands = capitano.state.commands.filter(
+		command => !command.hidden && !command.isWildcard(),
+	);
```
Comment (Page-) [inline lib/actions/help.js:83]: Changed this from
```ts
	const commands = _.reject(
		capitano.state.commands,
		command => command.hidden || command.isWildcard(),
	);
```
to this native version as the native version works properly with types

File: lib/actions/help.js [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+				console.log('\nGlobal Options:\n');
+				print(parse(capitano.state.globalOptions).sort());
+			}
+
+			return done();
+		})
+		.catch(done);
+};
+
+const commandHelp = (params, _options, done) =>
```
Comment (Page-) [inline lib/actions/help.js:132]: Renamed this to `commandHelp` to avoid linting issues

**Comment by srlowe** [comment]
Careful if you rebase after this PR merged, as exitWithExpectedError will have moved from `utils/patterns.ts` to `errors.ts`.  https://github.com/balena-io/balena-cli/pull/1791

**Comment by Page-** [comment]
@balena-ci rebase

**Comment by Page-** [comment]
It's not merged yet, but I think typescript would be able to catch that error in js :)

**Comment by Page-** [comment]
@balena-ci rebase

**Pull request merged**